app/core/constant.service.js

```diff
@@ -0,0 +1,11 @@
+(function(){
+  'use strict';
+  angular.module('meganote.notes').factory('noteConstants', noteConstants);
```

Comment:
I tried that, but angular threw an enormous fit. I decided to go with the factory/service idea, partially because I knew how to do it and partially because it can be later expanded to include other constants and globally useful functions if needed.